src/panels/lovelace/strategies/original-states-strategy.ts

```diff
@@ -38,16 +39,22 @@ export class OriginalStatesStrategy {
       subscribeEntityRegistry(hass.connection, () => undefined);
     }
 
-    const [areaEntries, deviceEntries, entityEntries, localize, energyPrefs] =
+    let energyPromise: Promise<EnergyPreferences> | undefined;
+
+    if (isComponentLoaded(hass, "energy")) {
+      energyPromise = getEnergyPreferences(hass);
+    }
+
+    const [areaEntries, deviceEntries, entityEntries, localize] =
       await Promise.all([
         subscribeOne(hass.connection, subscribeAreaRegistry),
         subscribeOne(hass.connection, subscribeDeviceRegistry),
         subscribeOne(hass.connection, subscribeEntityRegistry),
         hass.loadBackendTranslation("title"),
-        // It raises if not configured, just swallow that.
-        getEnergyPreferences(hass).catch(() => undefined),
       ]);
 
+    const energyPrefs = energyPromise ? await energyPromise : undefined;
```

Comment:
```suggestion
    const energyPrefs = isComponentLoaded(hass, "energy")
      ? await getEnergyPreferences(hass)
      : undefined;
```
Alternatively :shrug:

Reply:
We create the promise and start the fetching before we await 4 other promises. This way it's very likely to be finished when we await it here.